verl/trainer/config/ppo_trainer.yaml

```diff
@@ -506,11 +506,17 @@ actor_rollout_ref:
       enable: False
 
       # null for no limit (default max_length // 3)
-      max_turns: null
+      max_assistant_turns: null
```

Comment:
This looks like a breaking change, should be included in https://github.com/volcengine/verl/discussions/1902